Dockerfile

```diff
@@ -1,33 +1,27 @@
-FROM golang:1.13.5 AS base-env
+FROM alpine:3.13
 
-WORKDIR /src/
-
-COPY go.mod .
-COPY go.sum .
-RUN go mod download
-
-COPY . .
-
-FROM base-env AS build-env
-ENV BUILD_TARGET=/src/nri-kube-events
-ENV CGO_ENABLED=0
-RUN make compile
-
-FROM alpine:3.13 AS final
+# Set by docker automatically
+# If building with `docker build`, make sure to set GOOS/GOARCH explicitly when calling make:
+# `make compile GOOS=something GOARCH=something`
+# Otherwise the makefile will not append them to the binary name and docker build wil fail.
+ARG TARGETOS
+ARG TARGETARCH
 
 WORKDIR /app
 
 RUN apk add --no-cache --upgrade \
-    ca-certificates \
+    tini ca-certificates \
     && addgroup -g 2000 nri-kube-events \
     && adduser -D -H -u 1000 -G nri-kube-events nri-kube-events
-USER nri-kube-events
-
-COPY --from=build-env /src/nri-kube-events .
 EXPOSE 8080
 
+ADD --chmod=755 bin/nri-kube-events-${TARGETOS}-${TARGETARCH} ./
```

Comment:
does it work locally?

Reply:
Yes, both locally and afaik in the CI as well. Why do you ask?